src/pages/blog/components/_hero.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { TextWrapper, Title } from './_common';
+import device from 'themes/device.js'
+import { Flex } from 'components/containers'
+import { QueryImage } from 'components/elements'
+
+const MainWrapper = styled.div`
+    width: 100%;
+    background: black;
+    position: relative;
+    height: 600px;
+
+    .wrap {
+        max-width: 1440px;
+    }
+    .hero-content{
+        position: relative;
+        z-index: 99;
+    }
+
+    @media ${device.tablet} {
+        height: 400px;
+        text-align: left;
+        object-fit: cover;
+    }
+`
+const ImageWrapper = styled.div`
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+
+    @media ${device.tabletL} {
+        display: flex;
+    }
+`
+const Wrapper = styled(Flex)`
+    position: relative;
```

Comment:
```suggestion
    position: relative;
    max-width: 1440px;
    margin: 0 auto;
```

Reply:
Maybe we can set the Wrapper's max-width to 1440px as it stretched to the left side on bigger screen (>1600)

![image](https://user-images.githubusercontent.com/78890281/123041921-9fd13380-d428-11eb-9718-3eea6613d86e.png)

Reply:
Maybe let's keep it to 1600px to follow that of the header and footer.

Reply:
Hi @sean-binary @njazuli-deriv Discussed with Dennis, he want's this full width

Reply:
Nono, we meant the text container. Not the background image.